File: Python/ProjectEuler0001.py

# projecteuler.net/problem=1

# Answer: 233168

# github.com/firatksee


def multiples(limit):
    result = sum(i for i in range(limit) if i % 3 == 0 or i % 5 == 0)
    return result


if __name__ == "__main__":
    limit = 1000
    print(multiples(limit))

File: JavaScript/ProjectEuler0001.js
// projecteuler.net/problem=1

// Answer: 233168

// github.com/firatksee

function multiples(limit) {
    let result = 0;

    for (let i = 0; i < limit; i++) {
        if (i % 3 === 0 || i % 5 === 0) result += i;
    }

    return result;
}

console.log(multiples(1000));
